docs/contributor/03-40-kyma-bindings-processes.md

```diff
@@ -0,0 +1,75 @@
+# The Process of Creating a Kyma Binding
+
+The binding creation process, which starts with a PUT HTTP request sent to the `/oauth/v2/service_instances/{{instance_id}}/service_bindings/{{binding_id}}` endpoint, produces a binding with a kubeconfig that encapsulates a JWT token used for user authentication. The token is generated using Kubernetes TokenRequest attached to a ServiceAccount, ClusterRole, and ClusterRoleBinding, all named `kyma-binding-{{binding_id}}`. Such an approach allows for modifying permissions granted with the kubeconfig.
+Besides the kubeconfig, the response contains metadata with the **expires_at** field, which specifies the expiration time of the kubeconfig.
+To specify the duration for which the generated kubeconfig is valid explicitly, provide the **expiration_seconds** in the `parameter` object of the request body.
+
+The following diagram shows the flow of creating a service binding in Kyma Environment Broker. The process starts with a PUT request sent to KEB API.
+
+> [!NOTE]
+> On the diagram, "error" refers to a foreseen error in the process, not a server error.
+
+![Bindings Create Flow](../assets/bindings-create-flow.drawio.svg)
+
+The creation process is divided into three parts: configuration check, request validation, and binding creation.
+
+## Configuration Check
+
+If a feature flag for Kyma bindings is enabled, KEB does the following:
```

Comment:
Could this be: If you enable ... etc.?

Reply:
I think the feature flag is pre-configured in the environment. User would need to change the values in the chart to adjust the feature flag.